packages/web-client/tests/acceptance/deposit-withdrawal-test.ts

```diff
@@ -27,9 +28,9 @@ module('Acceptance | deposit and withdrawal', function (hooks) {
   });
 
   test('Depot and profile balances are listed when wallet is connected and update when the account changes', async function (this: Context, assert) {
-    let layer2Service: Layer2TestWeb3Strategy = this.owner.lookup(
-      'service:layer2-network'
-    ).strategy;
+    let layer2Service = (
+      this.owner.lookup('service:layer2-network') as Layer2Network
+    ).strategy as Layer2TestWeb3Strategy;
```

Comment:
While it's sometimes necessary to do this kind of aggressive casting, when it's necessary I think it belongs in a single documented utility function, not spread throughout the codebase.

But before you go changing these all, I'm going to send you a PR that extends the type of `owner.lookup` so it works the same as service injection, such that our `declare module '@ember/service' { ... }` stuff will inform it.

Reply:
https://github.com/cardstack/cardstack/pull/3211

Reply:
👍🏼